src/_posts/databases/2000-01-01-backup-policies.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Databases Backup Policies
+nav: Backup Policies
+modified_at: 2024-11-28 12:00:00
+tags: databases backup policy policies
+index: 2
+---
+
+
+Scalingo offers several options for backing up and restoring your data,
+depending on the service used. All backups are encrypted and securely stored in
+our infrastructure, within the same region as the source database.
```

Comment:
```suggestion
our infrastructure, within the same region as the source database, in three availability zones.
```

I think we could maybe share the blog post that talk about that in the doc also: https://scalingo.com/blog/data-resiliency-at-scalingo

Reply:
I'll add the availability zones mention.
IMHO, what's in the blog post should be somewhere in our documentation and we should probably link the documentation page then.

Would that be OK for you?

Reply:
Also, quick question: is the archive fully duplicated in these 3 zones?

Reply:
> Would that be OK for you?
Yes you're right

> Also, quick question: is the archive fully duplicated in these 3 zones?
By archive you mean logs archives?

Reply:
I meant the archive containing the backup :)

Reply:
> I meant the archive containing the backup :)

Yes, it's on the Object Storage. 2 zones are replicated on real-time. And the third one is replicated only every 24 hours